products/revenue_analytics/manifest.tsx

```diff
@@ -0,0 +1,38 @@
+import { IconRocket } from '@posthog/icons'
+import { urls } from 'scenes/urls'
+
+import { ProductManifest } from '../../frontend/src/types'
+
+export const manifest: ProductManifest = {
+    name: 'Revenue Analytics',
+    scenes: {
+        RevenueAnalytics: {
+            name: 'Revenue Analytics',
+            import: () => import('./frontend/RevenueAnalyticsDashboard'),
+            projectBased: true,
+            defaultDocsPath: '/docs/revenue-analytics',
+            activityScope: 'RevenueAnalytics',
+        },
+    },
+    routes: {
+        '/revenue_analytics': ['RevenueAnalytics', 'revenueAnalytics'],
+    },
+    redirects: {},
+    urls: {
+        revenueAnalytics: (): string => '/revenue_analytics',
+    },
+    fileSystemTypes: {
+        // TODO: add to project tree backend
+        // revenue_analytics: {
+        //     icon: <IconRocket />,
+        //     href: (ref: string) => urls.revenueAnalytics(ref),
+        // },
+    },
```

Comment:
I assume I dont need anything here, right @mariusandra

Reply:
You can leave this blank as I don't think you're adding new file types, but `treeItems` below should be `treeItemsNew` (adds to the context menu as "new X") or `treeItemsExplore` (not used now, but we'll replace the sidebar with these one day).

Reply:
Nice, thanks!

Suggestion: a nested `treeItems: { new: [], explore: [] }` sounds nicer :)

Reply:
It definitely does, and we might need even more. Luckily this is all easy to refactor once we actually start using this data in the sidebar.